src/config/sue/default.ts

```diff
@@ -55,6 +55,8 @@ const requiredFieldsDefaults = {
   }
 };
 
+const showFeedbackSection = false;
```

Comment:
this shouldn't be an extra variable just to hold the value false here, right?
can be `showFeedbackSection: false` directly in line 76